articles/java/sdk/http-client-pipeline.md

```diff
@@ -45,10 +45,18 @@ The following example shows you how to exclude the Netty dependency from a real
     </exclusions>
 </dependency>
 
+<!-- OkHttp -->
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-core-http-okhttp</artifactId>
-  <version>1.3.3</version>
+  <version>1.12.10</version>
+</dependency>
+
+<!-- JDK 11 HttpClient -->
```

Comment:
The JDK HttpClient was introduced in 11 but due to certain limitations with how HTTP headers are handled in JDK 11, we recommend using this library only with JDK 12+. So, in the comment, it would be better to mention that this is suitable only for JDK 12+ applications.

Reply:
There's an existing note above that it's only supported with 12+, I was matching another comment below. I could drop the 11 and keep it simply `JDK HttpClient` in both cases.

Reply:
That works. We can simply drop the version from this comment and the note above should be sufficient.

Reply:
@christarczon If you need help making this edit, please let us know.